include/nuttx/mutex.h

```diff
@@ -522,11 +523,11 @@ static inline int nxrmutex_breaklock(FAR rmutex_t *rmutex,
   pid_t tid = gettid();
   int ret = -EPERM;
 
-  if (rmutex->holder == tid)
+  if (rmutex->mutex.holder == tid)
```

Comment:
add nxmutex_breaklock?

Reply:
there is no count in struct mutex_s. 
I don't see the need to break lock.

Reply:
break is used to help caller leave the lock temp without pre knowledge whether the lock is held before. so, it's useful for no recurve case.